corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -436,7 +436,7 @@
 
     {% if not requirejs_main %}
       {% compress js %}
-        <script src="{% static 'select2/dist/js/select2.full.min.js' %}"></script>
+        <script src="{% static 'select-woo/dist/js/selectWoo.full.min.js' %}"></script>
```

Comment:
This file is the base template for most of HQ, you want `cloudcare/formplayer_home.html` to just update web apps.

Reply:
OK, I moved the import into `cloudcare/formplayer_home.html`.